ncs/ncs_scraper.py

```diff
@@ -0,0 +1,218 @@
+import configparser
+import json
+import logging
+import os
+import re
+import sys
+import time
+
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from bs4 import BeautifulSoup
+
+
+sys.path.insert(1,'../')
```

Comment:
Add a comment here saying why this is required. You can copy and paste from the comments you have made while committing

Reply:
Kirmi code was the one which was doing the taking the URL and converting it into soup. But now, I get the new page source in every iteration, and to convert that into soup, I used this lib here. ( It the most direct way possible)